setup.py

```diff
@@ -9,30 +9,38 @@ def read_file(filename):
     try:
         return open(filepath).read()
     except IOError:
-        return ''
+        return ""
 
 
 setup(
-    name='django-favicon',
-    version=__import__('favicon').__version__,
-    author='Evgeny Demchenko',
-    author_email='little_pea@list.ru',
+    name="django-favicon",
```

Comment:
In general, this change is great but why did you replace all single quotes with double?

Reply:
Oh. I globally use python-black on save. That's why single quotes turned to double automatically.